src/data/NotificationsEmailRepository.ts

```diff
@@ -30,9 +30,10 @@ export class NotificationsEmailRepository implements NotificationsRepository {
         log.info(`Send message (${subject}): ${notification.recipients.join(", ")}`);
 
         const res = await transport.sendMail({
-            to: notification.recipients,
+            to: process.env.RECIPIENTS ? process.env.RECIPIENTS.split(",") : notification.recipients,
```

Comment:
I've added this so we can test the full process without sending real emails to users, here you can put your email tot test.